### ISSUE TITLE
Optimizers in Machine Learning:

### ISSUE THREAD

**Issue by Anshika14528** [Recode-Hive/machine-learning-repos#19]
Explaining about various optimization techniques which is used to change model weights and learning rates, like Gradient Descent, Stochastic Gradient Descent, Stochastic Gradient descent with momentum, Mini-Batch Gradient Descent, AdaGrad, RMSProp, AdaDelta, and Adam. These optimization techniques play a critical role in the training of neural networks, as they help to improve the model by adjusting its parameters to minimize the loss of function value.

I agree to follow this project's Code of Conduct
I'm a GSSoC'24 contributor
I want to work on this issue

**Comment by sanjay-kv** [comment]
Make sure you create seperate folder and add the same

**Comment by lassmara** [comment]
Please assign me this issue.
